### DISCUSSION TITLE
OpenFGA `v1.7.0` has been released with experimental support for built-in access control 🔒 🎉

### DISCUSSION THREAD

**Discussion by rhamzeh** [openfga/community#382] [Announcements]
We've just released OpenFGA [v1.7.0](https://github.com/openfga/openfga/releases/tag/v1.7.0).

The highlights of this release are:
* Experimental support for [access control](https://openfga.dev/docs/getting-started/setup-openfga/access-control)
* Experimental optimizations to the `Check` endpoint
* Performance and caching improvements
* Various changes and fixes

Check out the [release notes](https://github.com/openfga/openfga/releases/tag/v1.7.0) for details

If you do end up trying our experimental access control or check optimization features, share your feedback below.

**Comment by xXAvoraXx** [comment]
Are the bugs related to nested check that I reported before included in the release? @rhamzeh 
https://github.com/openfga/openfga/security/advisories/GHSA-p4h9-5r9p-q938

**Comment by ddl-ebrown** [comment]
What is the current status of this feature? Is it still considered experimental 2 years after its initial introduction?

**Reply by aaguiarz** [reply]
Hi! Unfortunately yes. We did not have much feedback from companies using it yet, and we are not using this implementation internally yet.

**Reply by ddl-ebrown** [reply]
Thanks for the quick reply -- as we evaluate OpenFGA, it seems then like the simplest thing to do if we have internal service identities tied to our IdP is to front it with something like https://github.com/oauth2-proxy/oauth2-proxy or use a service mesh if the feature isn't yet production ready.

**Comment by aaguiarz** [comment]
You can do that yes, or you can try testing it and see how it works for you. Is not necessarily that is not 'production ready' but we are not aware of companies using it in production.
